### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by oliv3r** [stricaud/faup#117]
The last tag/release is from 2016, any plans to create a new release anytime?

**Comment by stricaud** [comment]
You are right, there has been enough improvement to have a new release. Thanks for  your comment! I will see what needs to be done right now to make it happen.
